spec/serializers/topic_list_item_serializer_spec.rb

```diff
@@ -122,7 +122,19 @@
       expect(json.key?(:op_can_like)).to eq(false)
     end
 
-    it "serializes op_liked" do
+    it "serializes op_can_like when plugin modifies the serialize_topic_op_likes_data to true" do
```

Comment:
I think for all the specs, a bit cleaner way to do this would be this style:

https://github.com/discourse/discourse/blob/b9f6a7e61dc37a8a812e30420f384f220dd8b01b/spec/serializers/topic_view_serializer_spec.rb#L657-L667

Reply:
Sounds good! Made it so 👍🏼